source/extensions/filters/network/zookeeper_proxy/decoder.h

```diff
@@ -117,15 +119,21 @@ using DecoderPtr = std::unique_ptr<Decoder>;
 
 class DecoderImpl : public Decoder, Logger::Loggable<Logger::Id::filter> {
 public:
-  explicit DecoderImpl(DecoderCallbacks& callbacks, uint32_t max_packet_bytes)
-      : callbacks_(callbacks), max_packet_bytes_(max_packet_bytes), helper_(max_packet_bytes) {}
+  explicit DecoderImpl(DecoderCallbacks& callbacks, uint32_t max_packet_bytes,
```

Comment:
if you remove `explicit` here I can close https://github.com/envoyproxy/envoy/pull/7828 :D